UiPath.FreeRdpClient/UiPath.FreeRdpClient/Internals/Logging.cs

```diff
@@ -1,17 +1,18 @@
-﻿using Microsoft.Extensions.Logging;
+﻿using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
 
 
 namespace UiPath.Rdp;
 
-internal static class Logging 
+internal class Logging : IHostedService
 {
     public static string ScopeName = "RunId";
-    internal static NativeInterface.LogCallback LogCallbackDelegate = Log;
-    private static NativeInterface.RegisterThreadScopeCallback RegisterThreadScopeCallbackDelegate = RegisterThreadScope;
+    private static NativeInterface.LogCallback LogCallbackDelegate = null!;
+    private static NativeInterface.RegisterThreadScopeCallback RegisterThreadScopeCallbackDelegate = null!;
```

Comment:
If the purpose was initially to avoid garbage collection, 
then these should become instance fields and get assigned in the instance constructor:

```csharp
    private readonly NativeInterface.LogCallback LogCallbackDelegate;
    private readonly NativeInterface.RegisterThreadScopeCallback RegisterThreadScopeCallbackDelegate;

   ...
   
    public Logging(ILoggerFactory loggerFactory)
    {
        LogCallbackDelegate = Log;
        RegisterThreadScopeCallbackDelegate = RegisterThreadScope;
    
        LoggerFactory = loggerFactory;        
    }
```